data/events.json

```diff
@@ -0,0 +1,492 @@
+  },
+  {
+    "eventDbId": "6000000066",
+    "studyDbId": "6000000010",
+    "eventDescription": "sprinkler irrigation after planting to facilitate seedling emergence.",
+    "date": {
+      "discreteDates": ["2017-04-21", "2017-04-22", "2017-04-23", "2017-04-24", "2017-04-25", "2017-04-26", "2017-04-27", "2017-04-28", "2017-04-29", "2017-04-30", "2017-05-01", "2017-05-02", "2017-05-03", "2017-05-04"]
+    },
+    "eventParameters": [
+        {
+          "code": "IROP",
+          "name": "irrigation_operation",
+          "value": "IR004",
+          "unit": "code",
+          "codeValueDescription": "Sprinkler"
+        },
+        {
+          "code": "IRRIG",
+          "name": "irrigation_applied",
+          "value": "y"
+        },
```

Comment:
according to ICASA, this should have `"unit": "code"` and probably a `"codeValueDescription": "Yes irrigation was applied"`